root/etc/rc.d/rc.local

```diff
@@ -5,7 +5,14 @@ exportKCL
 
 # setup root password
 if [[ ! -z "$KCL_FDI_ROOTPW" ]]; then
-  echo "root:$KCL_FDI_ROOTPW" | chpasswd >/dev/null
+  # remove possible double and/or single quotes surrounding password string
+  KCL_FDI_ROOTPW=`echo $KCL_FDI_ROOTPW | sed "s/\('\|\"\)//g"`
```

Comment:
Well, what you do is you strip all quotes from the string. Some people use these characters in passwords, when entered in plain text you break the feature. I understand the motivation, you can do it either if its a starting and ending quote (and only in case both are the same), or simply don't do it. I am for the latter.